libqtile/layout/xmonad.py

```diff
@@ -143,6 +143,7 @@ class MonadTall(_SimpleLayoutBase):
     _left = 0
     _right = 1
     _med_ratio = 0.5
+    _reset_ratio = 0.5
```

Comment:
Why another variable?

Reply:
@ramnes - the background to this is #2457.

You can probably get away with one variable if `_med_ratio` becomes configurable but there needs to be a check that it's between `min_ratio` and `max_ratio` (or it's automatically calculated to be the midpoint of these).

Reply:
When I use `cmd_reset` it reset all the layout including the master window but if I edit the `ratio` in the config file it keeps going back to `0.5`, I added this extra variable to avoid "issues" from the other functions variables "`min_ratio` and `max_ratio`" because they use `_med_ratio`